charts/stable/firefly-iii/values.yaml

```
@@ -0,0 +1,462 @@
+  DKR_BUILD_LOCALE: false
+
+  # Check if the SQLite database exists. Can be skipped if you're not using SQLite.
+  # Won't significantly speed up things.
+  DKR_CHECK_SQLITE: true
+
+  # Run database creation and migration commands. Disable this only if you're 100% sure the DB exists
+  # and is up to date.
+  DKR_RUN_MIGRATION: true
+
+  # Run database upgrade commands. Disable this only when you're 100% sure your DB is up-to-date
+  # with the latest fixes (outside of migrations!)
+  DKR_RUN_UPGRADE: true
+
+  # Verify database integrity. Includes all data checks and verifications.
+  # Disabling this makes Firefly III assume your DB is intact.
+  DKR_RUN_VERIFY: true
+
+  # Run database reporting commands. When disabled, Firefly III won't go over your data to report current state.
+  # Disabling this should have no impact on data integrity or safety but it won't warn you of possible issues.
+  DKR_RUN_REPORT: true
+
+  # Generate OAuth2 keys.
+  # When disabled, Firefly III won't attempt to generate OAuth2 Passport keys. This won't be an issue, IFF (if and only if)
+  # you had previously generated keys already and they're stored in your database for restoration.
+  DKR_RUN_PASSPORT_INSTALL: true
+
+  # Leave the following configuration vars as is.
+  # Unless you like to tinker and know what you're doing.
+  APP_NAME: FireflyIII
+  BROADCAST_DRIVER: log
+  QUEUE_DRIVER: sync
+  CACHE_PREFIX: firefly
+  PUSHER_KEY: ""
+  IPINFO_TOKEN: ""
+  PUSHER_SECRET: ""
+  PUSHER_ID: ""
+  DEMO_USERNAME: ""
+  DEMO_PASSWORD: ""
+  IS_HEROKU: false
+  FIREFLY_III_LAYOUT: v1
+
+  #
+  # If you have trouble configuring your Firefly III installation, DON'T BOTHER setting this variable.
+  # It won't work. It doesn't do ANYTHING. Don't believe the lies you read online. I'm not joking.
+  # This configuration value WILL NOT HELP.
+  #
+  # Notable exception to this rule is Synology, which, according to some users, will use APP_URL to rewrite stuff.
+  #
+  # This variable is ONLY used in some of the emails Firefly III sends around. Nowhere else.
+  # So when configuring anything WEB related this variable doesn't do anything. Nothing
+  #
+  # If you're stuck I understand you get desperate but look SOMEWHERE ELSE.
+  #
+  APP_URL: http://localhost
+
+# -- Configures service settings for the chart. Normally this does not need to be modified.
+# @default -- See values.yaml
+service:
+  main:
+    ports:
+      http:
+        port: 8080
+
+ingress:
+  # -- Enable and configure ingress settings for the chart under this key.
+  # @default -- See values.yaml
+  main:
+    enabled: false
+    hosts:
+      - host: chart-example.local
+        paths:
+          - path: /
+            pathType: Prefix
+            service:
+              port: http
+
+# -- Configure persistence settings for the chart under this key.
+# @default -- See values.yaml
+persistence:
+  config:
+    enabled: false
+  uploads:
+    enabled: true
```

```suggestion
    enabled: false
```
We usually leave persistance false as default and allow the users to chose to enable.